setup.py

```diff
@@ -72,9 +72,9 @@ def get_version():
             version += '.dev'
     if version.endswith(".dev"):
         init_sha = subprocess.getoutput(
-            "git log -n 1 --pretty=format:%H -- bindings/python/.pypi-version")
+            'git log -n 1 --pretty=format:%H -- bindings/python/.pypi-version')
         count = subprocess.getoutput(
-            "git rev-list --count HEAD \"^{}\"".format(init_sha))
+            'git rev-list --count HEAD \"^{}\"'.format(init_sha))
```

Comment:
If you change to single quotes, why are you still escaping double quotes?

Reply:
Thanks for pointing me on this. This slipped in this PR, so I tried now to get a more consistent use of single quotes. (And fixed this here.)